Library/Homebrew/cask/cask_loader.rb

```diff
@@ -51,10 +51,12 @@ def cask(header_token, **options, &block)
 
     # Loads a cask from a string.
     class FromContentLoader < AbstractContentLoader
+      sig {
```

Comment:
This matches the `try_new` `sig`s found throughout formulary.rb

Reply:
Suggestion: This should probably match the other type signatures for `try_new` in this file which also accept `Cask::Cask`.

```rb
      sig {
        override.params(ref: T.any(String, Pathname, Cask, URI::Generic), warn: T::Boolean)
                .returns(T.nilable(T.any(T.attached_class, FromAPILoader)))
      }
```

Reply:
That would make more sense, wouldn't it? 🙈